.env.template

```diff
@@ -0,0 +1,18 @@
+ATLASSIAN_USERNAME=
+ATLASSIAN_PASSWORD=
+JIRA_URL=
+BITBUCKET_URL=
+BAMBOO_URL=
+CONFLUENCE_URL=
+
+SPRING_DATASOURCE_URL=
+SPRING_DATASOURCE_USERNAME=
+SPRING_DATASOURCE_PASSWORD=
+
+SERVER_HOST=
+CLIENT_HOST=
+
+KEYCLOAK_HOST=
+KEYCLOAK_ISSUER_URI=
+KEYCLOAK_JWK_SET_URI=
+KEYCLOAK_CLIENT_SECRET=
```

Comment:
I would personally enter some sample values here, particularly for the URLs.

Reply:
done